examples/tensorflow-resnet/README.md

```diff
@@ -0,0 +1,187 @@
+
+```bash
+$ kubectl get po
+NAME                                READY   STATUS            RESTARTS   AGE                                                                                                                                                              
+tensorflow-resnet-bf8f6f5b6-22pgd   0/1     PodInitializing   0          7s                                                                                                                                                               
+tensorflow-resnet-bf8f6f5b6-59fq8   0/1     PodInitializing   0          7s                                                                                                                                                               
+tensorflow-resnet-bf8f6f5b6-qbd6v   0/1     PodInitializing   0          7s    
+```
+
+Once the model is loaded these pods are available to serve image classifications.
+```bash
+kubectl get po                                                                                                                                                                                         
+NAME                                READY   STATUS    RESTARTS   AGE                                                                                                                                                                      
+tensorflow-resnet-bf8f6f5b6-22pgd   1/1     Running   0          37s                                                                                                                                                                      
+tensorflow-resnet-bf8f6f5b6-59fq8   1/1     Running   0          37s                                                                                                                                                                      
+tensorflow-resnet-bf8f6f5b6-qbd6v   1/1     Running   0          37s
+```
+```bash
+$ kubectl get deployments
+NAME                READY   UP-TO-DATE   AVAILABLE   AGE
+tensorflow-resnet   3/3     3            3           10m
+```
+
+# Image classification
+
+Image classification requests should be done on a machine with Docker installed. Take note of one of the server's available IPs and confirm the available NodePort.  By default it should be 30090. After invoking you will recieve a result tensor.  
+The [./tensorClientResources/class.json](./tensorClientResources/class.json) has an array with the names of the results. Note that the classification in the JSON file is one number less then the result.  For example if you gave an image of a hammerhead shark you would
+receive the number 5 tensor result which is number 4 below.
+
+```json
+{  
+  "4": [
+    "n01494475",
+    "hammerhead"
+  ],
+  "5": [
+    "n01496331",
+    "electric_ray"
+  ]
+  }
+```
+
+A convience script is provided to provide the English name of the classification number.  See the example below.
```

Comment:
A convenience script is provided to map the classification number to an English description.